js/utils/InitModules/InitOOHazmat.js

```diff
@@ -1,8 +1,21 @@
 require("./InitOOUnderscore.js");
 
+var hazmatConfig = {};
+
+// 'debugHazmat' flag needs to be set before plugins are loaded. If we added
+// this flag to the OO namespace, it would be overriden during plugin initalization,
+// so we need to use a global var instead
+if (window && !window.debugHazmat) {
+  hazmatConfig = {
+    fail: function() { return; },
+    warn: function() { return; },
```

Comment:
Let's not overwrite fail and log for time being, they may convey important things, original request was to remove warnings, we'll see to it later if needed